### PR TITLE
Fixed readme and added node and bower modules in gitignore

### PR DESCRIPTION
**Pull request by itaditya**
I have changed the following in -
 1 README.md -
        * Spelling Mistakes.
         * Link for postgresql setup in Windows as I had trouble in that .
         * Clear the reason for two local servers.
 2 .gitignore -
            *Added entry for frontend node_modules and bower_components.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -74,3 +74,8 @@ ENV/
 
 # Fabric Files
 fabfile/
+
+# Frontend Modules
+node_modules
+frontend/bower_components
+frontend/node_modules
```
Comment (deshraj) [inline .gitignore]: @itaditya can you add a newline here?

Comment (aka-jain) [inline .gitignore]: @deshraj : we don't need these directories in main gitignore. We already have a different gitignore for frontend work.
@itaditya : please don't include any frontend related files/directories in main gitignore. Frontend specific gitignore is already present in frontend directory.

**Comment by aka-jain** [comment]
@itaditya Thanks for catching spelling mistakes :)

File: .gitignore (hunk 1 above)
Comment (itaditya) [inline .gitignore]: Ok Sir , I didn't knew that. I will fix this . 
So shall I add a commit on the branch from which I sent the PR and open a new PR ?

Comment (deshraj) [inline .gitignore]: You can fix this in the same PR.

**Comment by itaditya** [comment]
Sir do you you have a Slack Channel for this repo , where I can contact you

**Comment by deshraj** [comment]
@itaditya you can discuss on the gitter channel https://gitter.im/Cloud-CV/EvalAI

**Comment by itaditya** [comment]
Ok Sir :+1:

**Pull request merged**